wordpress-mobile/WordPress-iOS
# Remove the quick actions feature flag + unused code

Removes the Quick Actions feature flag.

Ref #13318

To test:
* Ensure that the Quick Actions view appears on the Blog Details page.
* Check that the buttons work as expected.

PR submission checklist:
